trino/sqlalchemy/compiler.py

```diff
@@ -147,6 +147,39 @@ def visit_BLOB(self, type_, **kw):
     def visit_DATETIME(self, type_, **kw):
         return self.visit_TIMESTAMP(type_, **kw)
 
+    def visit_INTERVAL(self, type_, **kw):
```

Comment:
I'm not really sure whether it's a correct approach. Interval in Trino has a different value and data type. 
For instance:
- data type: `INTERVAL YEAR TO MONTH`
- value: `INTERVAL '3' MONTH`

Reply:
Is this func supposed to return the data type or value? It seems like it returns a mix of both.